eng/pipelines/templates/steps/wrapped-use-python.yml

```diff
@@ -0,0 +1,26 @@
+parameters:
+  versionSpec: ''
+
+steps:
+  # use python 3.8 for tooling. packaging. platform.
+  - task: UsePythonVersion@0
+    displayName: "Use Python 3.8"
+    inputs:
+      versionSpec: 3.8
+
+  - pwsh: |
+      pip install packaging
+    displayName: Prep Environment
+
+  # select the appropriate version from manifest if present
+  - task: PythonScript@0
+    displayName: 'Check Version Spec Against Known Versions'
+    inputs:
+     scriptPath: 'scripts/devops_tasks/resolve_installer_location.py'
+     arguments: '${{ parameters.versionSpec }} --installer_folder="../_pyinstaller'
```

Comment:
Is it safe to assume that `../_pyinstaller` is the same place the UsePythonVersion task will look for the installed versions? Looks like it uses Agent.ToolsDirectory based on the docs at https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/use-python-version?view=azure-devops

Reply:
No! The installer actually unzips and installs into the precached tools directory! I'm just saving the installer to the disk so I can unzip it and run it. I owe this PR a rename run after I merged the script that installs into this script.